scripts/pylib/twister/twisterlib/testplan.py

```diff
@@ -952,3 +943,15 @@ def _create_build_dir_link(self, links_dir_path, instance):
         instance.build_dir = link_path
 
         self.link_dir_counter += 1
+
+
+def change_skip_to_error_if_integration(options, instance):
```

Comment:
looks like this function might be `class TestInstance` method

Reply:
@golowanow I think it is a good suggestion. I even wanted to apply this, but found the PR merged:) I assume that your comment was just a suggestion and hence no "+/-" review. Therefore, I will leave it as it is without creating a new PR (unless you object).